fatiando/mesher.py

```diff
@@ -1720,3 +1720,433 @@ def keep(cell):
             return True
         return False
     return [c for c in cells if keep(c)]
+
+
+class TriaxialEllipsoid (GeometricElement):
+
+    """
+    Create a triaxial ellipsoid.
+
+    .. note:: The coordinate system used is x -> North, y -> East
+    and z -> Down
+
+    Parameters:
+
+    * x, y, z : float
+        The coordinates of the center of the ellipsoid
+    * a, b, c : float
+        Semi-axes of the ellipsoid (a>b>c)
+    * alpha, gamma, delta
+        Orientation angles of the ellipsoid
```

Comment:
Same as the comment about the axes names. Use something more informative like dip, inclination, azimuth, etc.